src/libsync/configfile.cpp

```diff
@@ -258,6 +259,12 @@ int ConfigFile::timeout() const
     return settings.value(QLatin1String(timeoutC), 300).toInt(); // default to 5 min
 }
 
+bool ConfigFile::overrideServerChunkSize() const
+{
+    QSettings settings(configFile(), QSettings::IniFormat);
+    return settings.value(QLatin1String(overrideServerChunkSizeC), false).toBool();
+}
+
```

Comment:
is this really needed ?
that kind of defeat the purpose of having the max chunk size be provided by the server if the client is free to ignore it

Reply:
not needed, just wanted to make sure there was still a way to apply the config values even if the server is providing chunk sizing

I can remove